chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -715,14 +716,14 @@ where
         // take the biggest period possible.
         let period = BlockHashCount::get()
             .checked_next_power_of_two()
-            .map(|c| c / 2)
+            .map(|c| c.saturating_div(2))
             .unwrap_or(2) as u64;
 
         let current_block = System::block_number()
             .saturated_into::<u64>()
             // The `System::block_number` is initialized with `n+1`,
             // so the actual block number is `n`.
-            .saturating_sub(1);
+            .saturating_less_one();
```

Comment:
I hate that that this function exists, but whatever, there must be some reason for it

Reply:
Reverted in https://github.com/moondance-labs/tanssi/pull/969/commits/784b13e0eec61e060080f45b8ded9f3093f0b998